http/swagger.yml

```diff
@@ -6753,6 +6860,64 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/ScraperTargetResponse"
+    DocumentMeta:
+      type: object
+      properties:
+        name:
+          type: string
+        version:
+          type: string
+    Document:
+      type: object
+      properties:
+        id:
+          type: string
+          readOnly: true
+        meta:
+          $ref: "#/components/schemas/DocumentMeta"
+        content:
+          type: object
+        labels:
+          $ref: "#/components/schemas/Labels"
+    DocumentCreate:
+      type: object
+      properties:
+        meta:
+          $ref: "#/components/schemas/DocumentMeta"
+        content:
+          type: object
+        org:
+          type: string
+        orgID:
+          type: string
+        labels:
```

Comment:
would be good to add a comment on labels that they are strings of labels that will be attached to this object.

Also, mention that only one of org/orgID is required.